src/dmd/lib/package.d

```diff
@@ -1,13 +1,13 @@
 /**
  * A module defining an abstract library.
- * Implementations for various formats are in separate `libXXX.d` modules.
+ * Implementations for various formats are in `src/dmd/lib/` modules.
```

Comment:
Well technically that's not quite accurate:
- There is a 1-to-1 mapping between modules and files, but they are not the same concept. When referring to modules, we use the name in the `ModuleDeclaration`, e.g. `dmd.lib.elf`;
- `dmd.lib` still contains `scan*` which are not implementation, more like helpers;

But since it's consistent with the previous wording, we can fix it later. I think we have an opportunity to remove the function-backed factory now that we are going with proper encapsulation.